po/it.po

```diff
@@ -458,42 +450,39 @@ msgstr "Incolla inserendo pagine _pari"
 
 #: data/menu.ui.h:27
 msgid "Insert Blan_k Page"
-msgstr ""
+msgstr "Inserisci pagina vuota"
 
 #: data/menu.ui.h:28
 msgid "_Select"
-msgstr ""
+msgstr "_Seleziona"
 
 #: data/menu.ui.h:29
 msgid "Select _All"
-msgstr ""
+msgstr "_Seleziona_tutto"
```

Comment:
I think this is a typo, there should only be one accelerator (_). Also note the next comment about spaces.